comps/llms/text-generation/vllm/docker/Dockerfile.microservice

```diff
@@ -1,9 +1,10 @@
-
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
 FROM langchain/langchain:latest
 
+ARG ARCH="cpu"  # Set this to "cpu" or "gpu"
```

Comment:
If so, we need to maintain many versions of such Dockerfile, I suggest that there should be an environment variable to hold the value, and user can pass the specified ARCH value when building an image. Does it make sense?

Reply:
I suggested this as a quick workaround, but doing this right needs to happen across board and for all Dockerfiles in all repositories.
That can be worked on but it's going to a RFC and needs some discussion first.